plaidml/edsl/tests/edsl_error_test.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Intel Corporation
```

Comment:
Could this get merged into the `edsl_test.cc`, or were you thinking that one is already too long? I'm fine to start splitting up the source file I guess, but I want to have a single binary instead of having lots of individual test executables.

Reply:
My plan was to add tests here for more errors, as I add the `source_location` functionality to more of the API, and yes I was thinking that might start to be a bit long to just include with `edsl_test.cc`. I'll see how long it actually ends up being, but yes I can merge the binaries.